Source/Scene/MetadataTableProperty.js

```diff
@@ -217,6 +217,22 @@ MetadataTableProperty.prototype.set = function (index, value) {
   set(this, index, value);
 };
 
+/**
+ * Returns the typed array containing the property values.
```

Comment:
```suggestion
 * Returns a typed array containing the property values for a given propertyId
```

Reply:
I kept this as `Returns a typed array containing the property values` because this an operation on the property (no `propertyId` argument)